src/loadMessages.js

```diff
@@ -0,0 +1,11 @@
+export default function loadMessages(lang) {
+  const wrapped = import(`./messages/${lang}`)
+
+  if (typeof window !== 'undefined') {
+    return wrapped.then(msgs => msgs.default)
+  }
+
+  /* eslint-disable global-require, import/no-dynamic-require, prefer-template */
+  // return Promise.resolve(require('./messages/' + lang).default)
+  return __webpack_require__(wrapped.resolve()).default
+}
```

Comment:
Interestingly resolving with different parameters to `resolve()` all have the same result:

```
  console.log("Resolved with lang:", lang, wrapped.resolve(lang))
  console.log("Resolved with lang 2:", lang, wrapped.resolve({lang}))
  console.log("Plain resolved:", wrapped.resolve())
```

I put this code into my `loadMessages()` and all return `14`.